### PR TITLE
stake button takes you to /stake

### PR DESCRIPTION
**Pull request by MaxMillington**

### CODE REVIEW

File: packages/web/components/stake/stake-onboarding.tsx [hunk 1]
```diff
@@ -94,11 +97,16 @@ export const StakeOnboarding: React.FC<StakeOnboardingProps> = ({
 
   const { stakingAPR } = useGetApr();
 
+  const handleClose = () => {
+    () => setShowStakeIntroModal(false);
```
Comment (DavideSegullo) [inline packages/web/components/stake/stake-onboarding.tsx:101]: Is that arrow function required? Can't we just call `setShowStakeIntroModal(false)`?

**Review by DavideSegullo — CHANGES_REQUESTED** [review]

**Comment by CryptoAssassin1** [comment]
Start staking translation is missing.
Screenshot from 2024-01-11 21-07-15

File: packages/web/components/stake/stake-onboarding.tsx (hunk 1 above)
Comment (MaxMillington) [reply packages/web/components/stake/stake-onboarding.tsx:101]: yes. copypasta mistake

Comment (MaxMillington) [reply packages/web/components/stake/stake-onboarding.tsx]: @DavideSegullo updated

**Review by DavideSegullo — APPROVED** [review]

**Comment by MaxMillington** [comment]
@CryptoAssassin1 Will do in separate PR

**Pull request merged**